src/components/common/navbar.tsx

```diff
@@ -5,142 +5,258 @@ import Image from 'next/image'
+                        alignItems: "center",
+                        justifyContent: "space-between",
+                        margin: "10",
+                    }}
+                >
+                    <Link href="/">
+                        <div className="cursor-pointer">
+                            <Image alt="PEMM Logo" src={logo} width={500} height={75} />
+                        </div>
+                    </Link>
+                    <Box
+                        sx={{
+                            display: "flex",
+                            alignItems: "center",
+                            justifyContent: "flex-end",
+                            flexGrow: 1,
+                        }}
+                    >
+                        <Box
                             sx={{
-                                display: { xs: 'block', md: 'none' },
+                                display: { xs: "none", md: "flex", lg: "flex" },
+                                "& .MuiButton-root": {
+                                    textTransform: "capitalize",
+                                    fontWeight: "bold",
+                                    fontSize: "1.2rem",
+                                },
+                                "& .MuiButton-root:not(:last-child)": {
+                                    mr: "3rem",
+                                },
                             }}
                         >
-                            <MenuItem key="paintings" onClick={handleCloseNavMenu}>
-                                <Link href='/paintings'>
-                                    <Typography textAlign="center" color="primary"> Paintings </Typography>
-                                </Link>
-                            </MenuItem>
-                            <Link href='/stories'>
-                                <MenuItem key="stories" onClick={handleCloseNavMenu}>
-                                    <Typography textAlign="center" color="primary"> Stories </Typography>
-                                </MenuItem>
+                            <Link href="/stories">
+                                <Button color="primary">Stories</Button>
                             </Link>
-                            <MenuItem key="Manuscripts" onClick={handleCloseNavMenu}>
-                                <Link href='/manuscripts'>
-                                    <Typography textAlign="center" color="primary"> Manuscripts </Typography>
+
+                            <Button color="primary" onClick={handleClickPaintings}>
+                                Paintings
+                            </Button>
+                            <Menu
+                                anchorEl={anchorElPaintings}
+                                open={Boolean(anchorElPaintings)}
+                                onClose={handleClose}
+                            >
+                                <Link href="/paintings/all">
```

Comment:
This should go to the already existing paintings page at `/paintings` also please confirm that we need a drop down for the paintings page, I don't remember seeing a design for that.

Reply:
Updated